### ISSUE TITLE
Delete twisted.spread.newjelly

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1831]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#1831|
|Type|task|
|Created|2006-06-18 21:04:15Z|

We never made the leap to newjelly.  I doubt we ever will.  Having copy of jelly.py for what seems like a relatively minor refactoring was a bad idea anyway.  Fixes for bugs copied into newjelly have been applied to jelly but not newjelly.  Let's toss this junk.

<details><summary>Searchable metadata</summary>

```
trac-id__1831 1831
type__task task
reporter__exarkun exarkun
priority__highest highest
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1150664655000000 1150664655000000
changetime__1151018505000000 1151018505000000
version__None None
owner__ 
cc__warner cc__glyph cc__itamarst cc__exarkun
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

Ready for review in [branches/remove-newjelly-1831](branches/remove-newjelly-1831)

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set owner to @exarkun|
|-|-|

+1 (after an agreement from glyph, as well)

The way I read it, you moved some stuff from test_newjelly into test_jelly and removed all the newjelly stuff. right?

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

Right.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|

(In [17352]) Merge remove-newjelly-1831

Author: exarkun
Reviewers: radix, glyph
Fixes [#1831](https://github.com/twisted/twisted/issues/1831)

Move all of the newjelly tests into the jelly test module and
get rid of the newjelly code.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
